sysutils/kubectl/Portfile

```diff
@@ -62,7 +69,7 @@ subport kubectl-1.10 {
 if {${subport} == ${name}} {
     PortGroup           obsolete 1.0
     replaced_by         ${latestVersion}
-    version             1.15.0
+    version             1.16.1
     revision            1
```

Comment:
The `revision` line needs to be reset to 0 when upgrading the `version`.

Reply:
Given that this sub-port is labelled obsolete and `replaced_by` the latest version, should it even be updated at all?